src/index.html

```diff
@@ -11,7 +11,148 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="surname">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            required
+          >
+        </label>
+        <label class="put">
+          Name:
+          <input
+            type="text"
+            name="name"
+            required
+          >
+        </label>
+        <label class="put">
+          How old are You?
+          <input
+            name="age"
+            type="number"
+            min="1" max="100"
+            value="12"
+          >
+        </label>
+        <label class="put">
+          Full date of birth:
+          <input
+            type="date"
+            name="dateOfBirth"
+            placeholder="дд.мм.рррр"
+          >
+        </label>
+        <label class="put">
+          I accept the term  of the agreement
+          <input
+            name="agree"
+            type="checkbox"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <label>
+          E-mail:
+          <input
+            autocomplete="off"
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="put">
+          Password:
+          <input
+            autocomplete="off"
+            type="password"
+            name="password"
+            minlength="8"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <label>
+          Do you love cats?
+          <input
+            type="radio"
+            name="choose"
+            value="Yes"
+          >
+          Yes
+          <input
+            type="radio"
+            name="choose"
+            value="No"
+          >
+          No
+        </label>
```

Comment:
cover these inputs into different labels to avoid this 

https://user-images.githubusercontent.com/70220667/229437613-c15163da-9a4a-456c-a2eb-68c60c69bda4.mov